### ISSUE TITLE
Accessibility p-panel advanced: Required ARIA parent role not present: tablist

### ISSUE THREAD

**Issue by Rusamson** [primefaces/primeng#10738]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[x] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```
**Current behavior**

When testing the accessibility using axe devtool or the Lighthouse tool, the button with the role=”tab” gets reported as a critical error because the tablist role is missing from it’s parent
Here is the button :
&lt;button type="button" pripple="" role="tab" id="p-panel-0-label" aria-controls="p-panel-0-content" aria-expanded="true"&gt;

**Expected behavior**

Add a role attribute with a value  “tablist” in the p-panel

**Minimal reproduction of the problem with instructions**

Test the advanced p-panel component using the axe devTool extension or the Lighthouse extension, you will get below  critical errors: 
- Ensures elements with an ARIA role that require parent roles are contained by them 
- Required ARIA parent role not present: tablist

**What is the motivation / use case for changing the behavior?**

Diversity & Inclusion : I am working on a system where one of the main objectives is providing fully accessible contents to all users with equal access to information and functionality.

**Please tell us about your environment:**

Windows, NPM, Angular, VSCode

Angular version: 
PrimeNG version: 12.2.1
Browser: [Chrome 94.0.4606.81 ]
Language: [ TypeScript 2.8.0 ]
Node (for AoT issues): node --version = v14.17.0

**Comment by yigitfindikli** [comment]
All accessibility issues will be fixed in 2022 Q1. Thanks for your report.
